src/View/Helper/CdnHelper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace BootstrapUI\View\Helper;
+
+/**
+ * CndHelper class to insert Bootstrap CDN css and javascript assests into layout.
+ *
+ */
+//class CdnHelper extends \Cake\View\Helper
+class CdnHelper extends \Cake\View\Helper\HtmlHelper
```

Comment:
Don't see the need to extend Html helper.  The helper can use `$helper = ['Htm']` instead.